.github/workflows/health.yaml

```diff
@@ -0,0 +1,8 @@
+name: Health
+on:
+  pull_request:
+    branches: [ main, master ]
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+jobs:
+  health:
+    uses: dart-lang/ecosystem/.github/workflows/health.yaml@main
```

Comment:
This might need

```
    permissions:
      pull-requests: write
```

See https://github.com/dart-lang/i18n/blob/main/.github/workflows/health.yaml